Android/SortableTableView/nuget/Xamarin.SortableTableView.nuspec

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Xamarin.SortableTableView</id>
+<<<<<<< HEAD
+    <title>SortableTableView for Xamarin.MonoAndroid40</title>
+=======
+    <title>SortableTableView for Xamarin.MonoAndroid44</title>
+>>>>>>> 7c768d8b8c48205fda6b5b21e845e0faf130c00a
+    <version>2.2.1</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>A simple TableView and an advanced SortableTableView providing a lot of customisation possibilities to fit all needs.</description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <projectUrl>https://go.microsoft.com/fwlink/?linkid=871869</projectUrl>
+    <licenseUrl>https://go.microsoft.com/fwlink/?linkid=871806</licenseUrl>
+  </metadata>
+  <files>
+    <file src="output/SortableTableView.dll" target="lib/Xamarin.Android" />
```

Comment:
This is the wrong target name - to should be MonoAndroid60 or MonoAndroid80, what ever the minimum version of android is